frontend/src/lib/components/icon/icons/TipIcon.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import type {IconProps} from '../Icon.type';
+  import {IconBase} from '../base';
+
+  type $$Props = IconProps;
+  export let color: $$Props['color'] = undefined;
+  export let title: $$Props['title'] = $_('icons.tip');
+
+  // Icon path from Material Symbols Outlined ('FILL' 1, 'wght' 400, 'GRAD' 0, 'opsz' 24)
+  // https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Alightbulb%3AFILL%401%3Bwght%40400%3BGRAD%400%3Bopsz%4024
+  // License: Apache V 2.0, https://github.com/google/material-design-icons/blob/master/LICENSE
+</script>
+
+<!--
+@component
+An icon component.
+
+Use the properties to set the size of the icon, the color and the `desc` and 
+`title` of the icon. You can also pass any valid attributes to the `svg` 
+element.
+
+### Properties
+
+- See `$lib/components/icon/base/IconBase.type.ts` or code completion info.
+
+### Usage
+
+```tsx
+  <FooIcon color="primary" aria-hidden="true"/>
+```
+-->
+
+<IconBase fill={color} {title} viewBox="0 -960 960 960" {...$$restProps}>
+  <path
```

Comment:
Yeah, it's a bit problematic, because if we use a function that selects the path (but does not `import` it from an external file) then we always need to load the data for all of the icons when we use one of them. On the other hand, if we use a dynamic `import`, then there's popup.

The fact that there's some copied code doesn't seem such a huge issue for me, though, because it's very easy to use find-and-replace if there are any changes that need to be applied globally.

This said, we can go either way, of course. The popup issue can probably be relieved a bit by just importing the paths inside the `svg` element, so that it has a set size even while the `import` is still in progress.

Reply:
FYI this multi-file approach was modelled after [Svelte Material icons](https://github.com/ramiroaisen/svelte-material-icons/tree/master/svelte-material-icons).

Reply:
Okay, let's ditch individual components in that case and just make sure the size of the icon container stays the same during lazy loading.

As for maintaining the SVG files, I wouldn't auto-update those. We actually don't want them necessarily to be up-to-date with their original libraries (Material icons in this case for most of them), because if their style changes, it might no longer fit the design of the app or that of the custom icons.

We can get the SVGs easily from [this repo](https://github.com/marella/material-design-icons/blob/main/svg/round/arrow_circle_right.svg), for instance.

So, let's make this so that there's just one `Icon` component, which loads the necessary SVG file based on the icon name. Maybe we also ditch support for separate `fill` and `stroke` colours as well different `viewBox` sizes, because otherwise we would need to maintain a list of those info per icon separately as well.

I'll update this PR with such a version.